platform/mellanox/mlnx-platform-api/sonic_platform/device_data.py

```diff
@@ -0,0 +1,134 @@
+DEVICE_DATA = {
+    'ACS-MSN2700': {
```

Comment:
seems like for the different SKU which are different flavour of the same Platform type we need to define the same table over and over. can you please find a better way to do so?
for example: ACS-MSN2700, LS-SN2700, Mellanox-SN2700, Mellanox-SN2700-D48C8, etc.
soon when we will have a dynamic SKU creator we will not be able to support it thus it cannot be based on SKU.
we have the same limitation in sfputil and this is wrong.